javascript-functions/main.js

```diff
@@ -0,0 +1,34 @@
+function convertMinutesToSeconds(minutes) {
+  var seconds = minutes * 60;
+  return minutes + ' minutes is ' + seconds + ' seconds.';
```

Comment:
Close! If you look at the exercise it has an expected out put that is just a number. You don't need to worry about returning a label as we might do with console logs. You only need to return `seconds`